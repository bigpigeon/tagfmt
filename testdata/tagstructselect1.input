//tagfmt -sp "^User$" -f "json=snake(:field)"
package main

type User struct {
	Name     string ``
	Password string ``
	City     string ``
	State    string ``
}

type UserDB struct {
	ID       int    `toyorm:"primary key;auto_increment"`
	Name     string `toyorm:"type:VARCHAR(1024)"`
	Password string `toyorm:"type:VARCHAR(1024)"`
	City     string `toyorm:"type:VARCHAR(1024)"`
}