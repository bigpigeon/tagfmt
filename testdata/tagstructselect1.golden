//tagfmt -sp "^User$" -f "json=snake(:field)"
package main

type User struct {
	Name     string `json:"name"`
	Password string `json:"password"`
	City     string `json:"city"`
	State    string `json:"state"`
}

type UserDB struct {
	ID       int    `toyorm:"primary key;auto_increment"`
	Name     string `toyorm:"type:VARCHAR(1024)"`
	Password string `toyorm:"type:VARCHAR(1024)"`
	City     string `toyorm:"type:VARCHAR(1024)"`
}
